app/views/alchemy/admin/resources/_form.html.erb

```diff
@@ -8,6 +8,11 @@
         input_html: {class: 'alchemy_selectbox'} %>
     <% elsif attribute[:type].in? %i[date time datetime] %>
       <%= f.datepicker attribute[:name], resource_attribute_field_options(attribute) %>
+    <% elsif attribute[:enum].present? %>
+      <%= f.input attribute[:name],
+        collection: attribute[:enum],
```

Comment:
Should we support translating the enums?

Reply:
I think Rails has no built-in feature or convention how enums should be translated. Are you talking about the enum itself or its values, or both?

I've even seen some gems pop up and some people blogging about it. For me, the most obvious approach uses the I18n activerecord attributes structure for translating enum values.

Lets see an example:

```
de:
  activerecord:
    attributes:
      event:
        event_type: Veranstaltungsart
        event_types:
          conference: Konferenz
          expo: Ausstellung
```

Still, this is opinionated and forces Alchemy users into this structure. I am fine with it though.

Thoughts?

Reply:
I was thinking of just calling `pluralize` on the enum name in order to translate the values, but we might run into edge cases with words having equal singular and plural forms.

The predestined enum name "status" will work fine as Rails pluralises status with statuses, but there might be others that I can not think of right now.

We could also add a suffix "_values" instead.

```
de:
  activerecord:
    attributes:
      event:
        event_type: Veranstaltungsart
        event_type_values:
          conference: Konferenz
          expo: Ausstellung
```

Reply:
I went with the second (event_type_values) approach.

Reply:
Sounds good